packages/nc-gui/components/virtual-cell/components/ListChildItems.vue

```diff
@@ -233,7 +242,7 @@ onKeyStroke('Escape', () => {
       </div>
     </template>
     <div
-      v-else
+      v-if="!isDataExist && !isChildrenLoading"
```

Comment:
to make it more clean, you can just do in this way

```
v-else-if="!isChildrenLoading"
```

also you may think about the case like when `isDataExist` is `false` but `isChildrenLoading` is true.